system/jlib/jexcept.cpp

```diff
@@ -1258,14 +1284,17 @@ void jlib_decl enableSEHtoExceptionMapping()
     enableThreadSEH();
     SEHrestore = EnableSEHtranslation();
 #else
-    struct sigaction act;
     sigset_t blockset;
     sigemptyset(&blockset);
+    sigaddset(&blockset, SIGINT);
+    sigaddset(&blockset, SIGQUIT);
+    sigaddset(&blockset, SIGTERM);
```

Comment:
So this prevents these signals being issued again, when this thread is currently handling them, is that correct?
IOW it prevents re-entrantcy of termhandler ?